File: Python/PRAK601-2210817310017-Rohid Akmal.py
k, b = map(int, input().split())
angka = list(map(int, input().split()))
matrik = []
a = 0
for i in range(k) :
    matrik.append(angka[a : a + b])
    a = a + b
for k in matrik :
    for angka in k :
        print(angka, end=' ')
    print()

File: C/PRAK601-2210817310017-Rohid_Akmal.c
#include <stdio.h>

void main(){
    int b, k, i, j, in;
    scanf("%d %d", &b, &k);
    int matrik[b][k];
    for (i = 0; i < b; i++){
        for (j = 0; j < k; j++){
        scanf("%d", &in);
        matrik[i][j] = in;}
    }
    printf("\n");
    for (i = 0; i < b; i++){
        for (j = 0; j < k; j++){
            printf("%d ", matrik[i][j]);}
        printf("\n");}
}